GenderPayGap.WebUI/Views/Register/OrganisationType.cshtml

```diff
@@ -31,24 +31,57 @@
                         @Html.ValidationMessageFor(model => model.SectorType, null, new {@class = "error-danger"})
                     </legend>
 
+                    <div class="multiple-choice">
+                        <input id="public" type="radio" name="SectorType" value="@(SectorTypes.Public)">
+                        <label for="public">
+                            Public sector organisation
+                        </label>
+                    </div>
                     <div class="multiple-choice">
                         <input id="private" type="radio" name="SectorType" value="@(SectorTypes.Private)">
                         <label for="private">
-                            Private limited company,<br/>
-                            Limited liability partnership,<br/>
-                            Charity,<br/>
+                            Private limited company
+                        </label>
+                    </div>
+                    <div class="multiple-choice">
+                        <input id="publicLimitedCompany" type="radio" name="SectorType" value="@(SectorTypes.Private)">
+                        <label for="publicLimitedCompany">
                             Public limited company
                         </label>
                     </div>
                     <div class="multiple-choice">
-                        <input id="public" type="radio" name="SectorType" value="@(SectorTypes.Public)">
-                        <label for="public">
-                            Public sector organisation
+                        <input id="limitedLiabilityPartnership" type="radio" name="SectorType" value="@(SectorTypes.Private)">
+                        <label for="limitedLiabilityPartnership">
+                            Limited liability partnership
                         </label>
                     </div>
+                    <div class="multiple-choice">
+                        <input id="charity" type="radio" name="SectorType" value="@(SectorTypes.Private)">
+                        <label for="charity">
+                            Charity
+                        </label>
+                    </div>
+
                 </fieldset>
             </div>
-            <br/>
+            @(Html.GovUkInsetText(new InsetTextViewModel

```

Comment:
Whilst I'm happy for you to use `Html.GovUkInsetText`, the HTML equivalent is very simple, so feel free to use that if you prefer.

I think it's worthwhile using the C# components for any complicated / data-driven components, but for plain / styling components, I'd happy for you to use plain HTML if that makes it clearer.

Reply:
I don't think this view is using the new `GovUkFrontendLayout` so the stylesheets aren't being pulled through. This means that adding `class="govuk-inset-text"` doesn't give us the inset styling. So I used this component as a temporary work-around until we update this page.